index.js

```diff
@@ -1,130 +1,128 @@
-var _ = require('lodash');
+'use strict';
+
 var Q = require('q');
 var npmKeyword = require('npm-keyword');
 var packageJson = require('package-json');
 var ghGot = require('gh-got');
 var EventEmitter = require('events').EventEmitter;
 
-var UPDATE_INTERVAL_IN_SECONDS = 60 * 60;
+var UPDATE_INTERVAL_IN_SECONDS = 60 * 60 + 1;
```

Comment:
I assume it's padded to make sure we're on a new rate limit hour. I would make that `1` => `10` just to be sure.